packages/web-frontend/src/projects/sagas.js

```diff
@@ -24,14 +25,26 @@ export function* fetchProjectData() {
 
 export function* fetchCustomLandingPageData() {
   try {
+    yield put({
+      type: CUSTOM_LANDING_PAGE_LOADING,
+      isLoading: true,
+    });
     const { routing: location } = yield select();
 
     const data = yield call(request, `landingPage/${location.pathname.substring(1)}`);
     yield put({
       type: SET_CUSTOM_LANDING_PAGE_DATA,
       data,
     });
+    yield put({
+      type: CUSTOM_LANDING_PAGE_LOADING,
+      isLoading: false,
+    });
   } catch (error) {
+    yield put({
+      type: CUSTOM_LANDING_PAGE_LOADING,
+      isLoading: false,
+    });
     // eslint-disable-next-line no-console
     console.error(error);
```

Comment:
Potentially if you wanted to be extra fancy you could use a `finally` instead of repeating the same call twice, but this is very minor